### ISSUE TITLE
Targeting Windows user account folders for output path should fail config validation

### ISSUE THREAD

**Issue by yezr** [BrightSpots/rcv#757]
*This issue is directly related to #748 and the 1.3.2 build. Along with #747, this should be part of the two-build system specific to CA*

On Windows, one of the requirements for ensuring read-only access for RCTab output is setting the right folder permissions on the output folder RCTab is configured for. Step by step instructions for setting the necessary folder permissions are described in the installation instructions of Section 22 of the TDP. 

From testing in Windows, I found that #748 successfully prevents modifications, but user accounts retain delete and create permissions to their user account folders (e.g. Desktop, My Documents) in `c:\users\` despite following the installation instructions to restrict permissions on those folders. 

Can we add a check into the config verification step before tabulation begins to throw a halting error if the configured output path resolves to any path under `c:\users\`?

**Comment by yezr** [comment]
For the exception text, can we be explicit about why this is failing. Something along the lines of 

> To ensure read-only access to RCTab output files users **must not** set the output path to user account folders like My Documents or My Desktop - any path under `c:\users\`. Users must select a path outside these folders, for example `c:\RCTab`. Please select a different path in the Output Tab of the app.

**Comment by HEdingfield** [comment]
I'm struggling to understand the premise of this request: if the user has permissions to write files in C:\RCTab\output, then they also inherently have delete and create permissions there as well, no?

And if they don't have full access to a folder to do this, they won't be able to instruct RCTab to spit them out there in the first place.

I tested this out in Windows and was able to delete the files in the folder I created in C:\ without any problem (didn't even ask for sysadmin approval)

**Comment by artoonie** [comment]
You need to follow the instructions in Section 22 here: https://docs.google.com/document/d/1lJr-AYWN0yhWcclRuzcjsnXZOcrjWRQ7Zbh3YnCEVE0/edit

My understanding is that the steps in Section 22 do not work in your user directory, but would work in (e.g.) C:/output/

**Comment by yezr** [comment]
The main goal of 1.3.2 is to make it more programmatically difficult for an insider to maliciously alter RCTab input/output. One step to do that was to make RCTab output read only. I installed 1.4.0 which had the read-only files implemented but found the same catch-22 you mention @HEdingfield: write permissions also grant delete permissions. 

I tried everything I could to enforce true read-only permissions and the most approachable solution I found still requires procedural steps. Those procedural steps are listed most explicitly in the TDP in Section 22 - Installation Instructions for Windows OS. 

It requires two accounts - one Windows Administrator account and one 'RCTab' Windows standard account. During installation, Election admins explicitly deny write permissions to the output folder for an 'RCTab' Windows Standard account. During tabulation, the 'RCTab' Standard account is logged in and **only** the RCTab software only running as the Administrator account. This ensures the logged in user cannot delete RCTab output (if configured correctly) and cannot remove the read-only restrictions. 

This was the most approachable solution I could find without some type of admin/standard user system within RCTab itself to ensure read only output.
